### PR TITLE
Created tests for EthWrapper

### PR DESCRIPTION
**Pull request by nivgoldstein**

### CODE REVIEW

File: tests/test_eth_wrapper.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import pytest
+import time
+
+from wrap import EthWrapper
+
+PRIVATEKEY = '0xd5d2d701f19ffd8c9a0a0dd8518912498e3b0fcaa114256ce51226f3768aced5'
```
Comment (oleg-efrontier) [inline tests/test_eth_wrapper.py:6]: Do not store private keys in github, even in private repo.

File: tests/test_eth_wrapper.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ETH_WRAPPER = EthWrapper(PRIVATEKEY)
```
Comment (oleg-efrontier) [inline tests/test_eth_wrapper.py:7]: If we're using private key only to build wallet number, then it's better to use wallet number.
Place wallet number in testdata.ini fixture file and load it.
Do not put testdata.ini to git , use testdata.ini-example instead with dummy wallet number.

File: tests/test_eth_wrapper.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+def test_wrap():
+    wei = int(1e17)
+
+    before_eth = ETH_WRAPPER.get_eth_balance()
+    before_weth = ETH_WRAPPER.get_weth_balance()
+
+    ETH_WRAPPER.wrap(wei)
+
+    time.sleep(10)
```
Comment (oleg-efrontier) [inline tests/test_eth_wrapper.py:20]: If wrap/unwrap wait for transaction to finish, why additional sleep?

File: tests/test_eth_wrapper.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+REASONABLE_GAS = 1e14
+
+
+def test_wrap():
+    wei = int(1e17)
+
+    before_eth = ETH_WRAPPER.get_eth_balance()
+    before_weth = ETH_WRAPPER.get_weth_balance()
+
+    ETH_WRAPPER.wrap(wei)
```
Comment (oleg-efrontier) [inline tests/test_eth_wrapper.py:18]: Check that wrap/unwrap function accept amount in ether. ( change their arguments name  to clarify that).

File: tests/test_eth_wrapper.py (hunk 1 above)
Comment (nivgoldstein) [reply tests/test_eth_wrapper.py:6]: It's a private key that only contains rinkeby eth, but still, I'll put it in a text file

File: tests/test_eth_wrapper.py (hunk 4 above)
Comment (nivgoldstein) [reply tests/test_eth_wrapper.py:18]: They accept wei, and they're parameter is 'wei'

File: tests/test_eth_wrapper.py (hunk 2 above)
Comment (nivgoldstein) [reply tests/test_eth_wrapper.py:7]: By wallet number you mean wallet address?
We need the private key to sign the transaction, otherwise it won't work.

Comment (nivgoldstein) [reply tests/test_eth_wrapper.py]: Why ini?
I put in a JSON

File: tests/test_eth_wrapper.py (hunk 4 above)
Comment (oleg-efrontier) [reply tests/test_eth_wrapper.py]: These are high level functions, they need to accept eth/weth and do conversion inside. 

What was the reason you choose to change the specification?

File: tests/test_eth_wrapper.py (hunk 2 above)
Comment (oleg-efrontier) [reply tests/test_eth_wrapper.py]: Put it in Json if you want. The idea is the dame.

**Pull request merged**